src/ir_def/construct.ml

```diff
@@ -299,14 +309,17 @@ let callE exp1 typs exp2 =
     | T.Non -> T.Non
     | _ -> raise (Invalid_argument "callE expect a function")
   in
+  let p = CallPrim typs in
+  let es = [exp1; exp2] in
   { it = PrimE (CallPrim typs, [exp1; exp2]);
```

Comment:
`{ it = PrimE (p, es);`

Reply:
Agreed, fixing in #3Agreed, fixing in #3727727